src/api/system.c

```diff
@@ -1,4 +1,5 @@
 #include <SDL.h>
+#include <string.h>
```

Comment:
Had to add this because there was no definition of `strerror` so it defaulted to a return value of int, which caused segmentation faults since it returns a char*